schema.json

```diff
@@ -979,9 +993,69 @@
             "properties": {
                 "name": {
                     "type": "string"
+                },
+                "address": {
+                    "$ref": "#/definitions/address"
+                },
+                "alias": {
+                    "$ref": "#/definitions/alias"
+                },
+                "city": {
+                    "$ref": "#/definitions/city"
+                },
+                "country": {
+                    "$ref": "#/definitions/country"
+                },
+                "email": {
+                    "$ref": "#/definitions/email"
+                },
+                "fax": {
+                    "$ref": "#/definitions/fax"
+                },
+                "orcid": {
+                    "$ref": "#/definitions/orcid"
+                },
+                "post-code": {
+                    "$ref": "#/definitions/post-code"
+                },
+                "region": {
+                    "$ref": "#/definitions/region"
+                },
+                "tel": {
+                    "$ref": "#/definitions/tel"
+                },
+                "website": {
+                    "$ref": "#/definitions/url"
+                },
+                "date-start": {
+                    "$ref": "#/definitions/date"
+                },
+                "date-end": {
+                    "$ref": "#/definitions/date"
+                },
+                "location": {
+                    "type": "string"
```

Comment:
Somewhere around here there should be a `required` with `name` in it methinks

Reply:
Fixed in a162d1eb9b27e721b99fa73a23a4e4c26368acd1.